api/v1alpha1/nginxingresscontroller_types.go

```diff
@@ -61,6 +61,10 @@ type NginxIngressControllerSpec struct {
 	// +optional
 	DefaultBackendService *NICNamespacedName `json:"defaultBackendService,omitempty"`
 
+	// CustomHTTPErrors defines the error codes that the NginxIngressController should send to its default-backend in case of error.
+	// +optional
+	CustomHTTPErrors []int `json:"customHTTPErrors,omitempty"`
```

Comment:
this should be an int32

> Fields may use most of the primitive types.  Numbers are the exception:
for API compatibility purposes, we accept three forms of numbers: `int32`
and `int64` for integers, and `resource.Quantity` for decimals.

Reply:
https://book.kubebuilder.io/cronjob-tutorial/api-design.html?highlight=int32#designing-an-api